content/docs/command-reference/remote/modify.md

```diff
@@ -346,6 +346,19 @@ $ dvc push
 For more on the supported env vars, please see the
 [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-environment-variables)
 
+- `version_aware` - Use
+  [version-aware](/docs/user-guide/data-management/cloud-versioning#version-aware-remotes)
+  cloud versioning features for this S3 remote. This requires that
+  [S3 Versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html)
+  be enabled on the specified S3 bucket.
+
+- `worktree` - Use
+  [worktree](/docs/user-guide/data-management/cloud-versioning#worktree-remotes)
+  cloud versioning features for this S3 remote. This requires that
+  [S3 Versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html)
+  be enabled on the specified S3 bucket. When both `version_aware` and
+  `worktree` are set, `worktree` takes precedence.
```

Comment:
Does this mean that `worktree` alone is enough to set and that additionally setting `version_aware` is redundant?

Reply:
Yes, `worktree = true` now implies `version_aware = true`, so you no longer need to set both. If both `worktree = true` and `version_aware = false` are explicitly set you will get an error